### ISSUE TITLE
Add prove of App.net identities

### ISSUE THREAD

**Issue by liGhun** [keybase/keybase-issues#575]
App.net might a small, nerdy service those days but it would be great to verify my identies there here too.

Even more as some developers there (including myself) already think about combining the private messages with keybase.io based encryptions...

**Comment by blumenkraft** [comment]
i second that!

**Comment by sevan** [comment]
It would be useful

**Comment by Lechindianer** [comment]
That would be great, indeed!

**Comment by 33mhz** [comment]
Yeah, we're considering encryption projects for the next hackathon there.

I'm all for this.

**Comment by thedan84** [comment]
Think it would be useful, yes.

**Comment by wilfreddekok** [comment]
Agreed!

**Comment by jdscolam** [comment]
+1

**Comment by jconnary** [comment]
+1

**Comment by MisterPoppet** [comment]
+1. I don't use any of the other services.
